ArkEcosystem/payvo-wallet
# refactor: responsive portfolio breakdown

## Summary

Closes https://github.com/ArkEcosystem/payvo-wallet/issues/998

### How to test

An easy way to test with different values on the charts is to hardcode an array of values (otherwise you are going to need to have wallets in different networks with money)

To do that:
1. open the fike `src/domains/dashboard/hooks/use-portfolio-breakdown.ts`
2. Replace the `portfolioItems` assignation with this snippet (see the latest screenshot for a diff example), also feel free to update the percents manually
```php

		const portfolioItems = [
			{
				amount: 373.133_657_93,
				convertedAmount: 302.238_262_923_300_06,
				label: "ARK",
				percent: 50.5,
			},
			{
				amount: 222.133_657_93,
				convertedAmount: 22.238_262_923_300_06,
				label: "BLK",
				percent: 21.5,
			},
			{
				amount: 222.133_657_93,
				convertedAmount: 22.238_262_923_300_06,
				label: "BIND",
				percent: 15.6,
			},
			{
				amount: 222.133_657_93,
				convertedAmount: 22.238_262_923_300_06,
				label: "XQR",
				percent: 12.3,
			},
		];
```

https://user-images.githubusercontent.com/17262776/155387355-12ffb4c8-dc80-4cc3-96c6-8fe10722a2dc.mov

https://user-images.githubusercontent.com/17262776/155387370-fae0ab37-780f-4d7f-962c-9952ca39a667.mov

https://user-images.githubusercontent.com/17262776/155387373-54f5a71b-57ec-4728-9550-48a692ab4680.mov

https://user-images.githubusercontent.com/17262776/155387377-d1cb38d4-6278-4a35-b7f6-5ed6beaf0636.mov

![image](https://user-images.githubusercontent.com/17262776/155388034-edc67dcb-91cf-4f03-9e01-d0f14d237779.png)








## Checklist
